docs/install/add-site.md

```diff
@@ -0,0 +1,76 @@
+---
+myst:
+  html_meta:
+    "description": "How to add a Plone site to an existing Zope instance"
+    "property=og:description": "How to add a Plone site to an existing Zope instance"
+    "property=og:title": "Add a Plone site"
+    "keywords": "Plone 6, create, add, factory, distributions"
+---
+
+% TODO: This should move into the Admin guide once https://github.com/plone/documentation/pull/1746 is merged.
+
+(add-a-plone-site-label)=
+
+# Add a Plone site
+
+This section explains how to add a Plone site to an existing Zope instance.
```

Comment:
"Zope instance" is a term that comes out of nowhere and is not defined in the glossary. Do we have a better term? Can we call it a "Plone instance"? The glossary contains the phrase "Zope WSGI instance" in two of its definitions, but is not defined itself.

Reply:
I think "Zope instance" is probably a concept that's worth including in the glossary.

"Plone instance" could be confusing, since it's possible to add multiple Plone sites in the same Zope instance.

"Zope WSGI instance" just sounds weird to me. WSGI is one way (now the only way) to publish a Zope instance over HTTP, but it's no longer a special kind of Zope instance and it's certainly not a brand.

Reply:
Done in 9267edbb925208f4fee7569cc06fc52913c1a732.

We use "Plone instance" and "Zope / Plone instance" in several places throughout the docs. I created a new issue to handle that: https://github.com/plone/documentation/issues/1751